docs/guide/input-validation.md

```diff
@@ -372,6 +373,14 @@ class MyForm extends Model
 }
 ```
 
+> Note: you can use [[yii\validators\InlineValidator::addError()]] for adding errors instead. That way the error
+> message can be formatted using [[yii\i18n\I18N::format()]] right away. Use `{attribute}` and `{value}` in the error
+> message to refer to attribute label (no need to get it manually) and attribute value accordingly:
```

Comment:
to refer to **an** attribute label (please mind these words before the subject, like `a` `an`, `the` etc...